rest_framework/throttling.py

```diff
@@ -101,9 +102,26 @@ def parse_rate(self, rate):
         """
         if rate is None:
             return (None, None)
-        num, period = rate.split('/')
-        num_requests = int(num)
-        duration = {'s': 1, 'm': 60, 'h': 3600, 'd': 86400}[period[0]]
+
+        try:
+            num, period = rate.split('/')
+            num_requests = int(num)
+            period_timescale = re.findall(r'^\d*[s|m|h|d]', period)[0]
```

Comment:
I guess allow optional whitespace between the number and the interval.

Reply:
Added.